x-pack/legacy/plugins/infra/public/components/source_configuration/log_columns_configuration_panel.tsx

```diff
@@ -56,12 +61,32 @@ export const LogColumnsConfigurationPanel: React.FunctionComponent<
       </EuiFlexItem>
     </EuiFlexGroup>
     {logColumnConfiguration.length > 0 ? (
-      logColumnConfiguration.map((column, index) => (
-        <LogColumnConfigurationPanel
-          logColumnConfigurationProps={column}
-          key={`logColumnConfigurationPanel-${index}`}
-        />
-      ))
+      <EuiDragDropContext
+        onDragEnd={({ source, destination }) =>
```

Comment:
How about we move this function up a bit and memoize it using `useCallback` to avoid passing a changing handler function to the `EuiDragDropContext` all the time?